CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Extends statement redaction to support `INSERT/REPLACE/UPSERT INTO`. 
```

Comment:
Nit -- can you add this to one of the below sections? Added/fixed?

Reply:
Ah...that was miss, will do.